### PR TITLE
feature/cisco_ios_add_fields

### PR DESCRIPTION
**Pull request by clement-burtscher-sekoia**
## Description

Relates to https://github.com/SekoiaLab/integration/issues/1685

### Added

- Parse ECS dynamic fields for HTTPS connection logs without Cisco header:
    - destination.ip
    - event.action
    - event.category
    - event.code
    - event.outcome
    - event.reason
    - event.type
    - network.protocol
    - source.ip
    - user.name
- Add smart descriptions for HTTPS connection accepted and terminated events

### Changed

- Update parser fallback to handle description-only messages and to map HTTPS connection states to ECS

## Type of Change

- [ ] New intake format
- [x] Update to existing intake format
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product: Cisco/cisco-ios

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`uv run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`uv run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Extend Cisco IOS parser to handle HTTPS connection logs without Cisco headers and map them to appropriate ECS network event fields.

New Features:
- Extract ECS fields such as source and destination IPs, user name, protocol, action, type, outcome, and code from HTTPS connection messages, including those without Cisco headers.
- Provide smart descriptions for HTTPS connection accepted and terminated events.

Enhancements:
- Update parser fallback logic to use either structured description or raw description and to derive ECS fields from HTTPS connection state.
- Document the new HTTPS connection handling in the Cisco IOS changelog.

Tests:
- Add tests covering HTTPS connection accepted and terminated events to validate the new parsing and smart descriptions.

### CODE REVIEW

File: Cisco/cisco-ios/CHANGELOG.md [hunk 1]
```diff
@@ -6,3 +6,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [Unreleased] - 2026-06-16
+
+### Added
```
[the remark this answers is not in the document]
Comment (clement-burtscher-sekoia) [reply Cisco/cisco-ios/CHANGELOG.md:12]: Tags cannot be determined before deployment

[the remark this answers is not in the document]
Comment (clement-burtscher-sekoia) [reply Cisco/cisco-ios/CHANGELOG.md:12]: Tags cannot be determined before deployment

**Review by mchupeau-sk — APPROVED** [review]
LGFM
